### PR TITLE
Vcon 44 Add Botframework Service sample code

### PR DESCRIPTION
**Pull request by neilmispelaar**
### TLDR

Successfully added `directlinejs` dependency to the project. It works now, you can create a `directlinejs` object and send / receive new activities. The culprit was the `package-lock.json` file and the existing npm dependency tree. Removing the offending file and re-running `npm install` fixed everything. Feel free to use this pull request or reject it and steal the code. Up to you. 

### Description

We were previously trying to add `directlinejs` to the project and the stupid thing was experiencing a tonne of errors. Experiments were done with older versions of directline, to mixed success. Thomas was able to test in a next.js project and it worked fine. I was sad vite was letting me down, but figured it might be the lack of webpack specific bundling features. 

Anyways, to my surprise I tested in a new vite project and it worked fine. Then tested in the sample rolodex app and it worked fine. So I assumed the issue was localised to the project. Then blew away the `package-lock.json` file and updated all the dependencies (vite, vuex, vue, and tailwind were also all outdated). Re ran `npm install` and tested the sample code in the service and everything worked fine. 

### Whats in here

- Updated vite, vuex, vue, tailwindcss dependencies 
- Added directlinejs dependency
- Added sample code to demonstrate the directline object working fine

### CODE REVIEW

**Review by nibivi77 — APPROVED** [review]
Thank you so much Neil! This is great! worked as expected on my end!
